src/devtools/client/webconsole/reducers/messages.ts

```diff
@@ -285,6 +282,26 @@ const messagesSlice = createSlice({
         state as MessageState
       );
     },
+    clearMessages(state) {
+      // Note that we don't use removeMessagesFromState() in this case, because we're removing ALL messages.
+      // That method does extra work to handle the case where we are only removing SOME messages.
+      messagesAdapter.removeAll((state as MessageState).messages);
+
+      state.filteredMessagesCount = getDefaultFiltersCounter();
+      state.messagesLoaded = false;
+      state.messagesUiById = [];
+      state.overflow = false;
+      state.visibleMessages = [];
+    },
+    setConsoleOverflowed(state, action: PayloadAction<boolean>) {
+      state.overflow = action.payload;
+    },
+    setLastFetchedForFocusRegion(state, action: PayloadAction<FocusRegion | null>) {
+      state.lastFetchedForFocusRegion = action.payload;
+    },
+    setMessagesLoaded(state, action: PayloadAction<boolean>) {
+      state.messagesLoaded = action.payload;
```

Comment:
:question: I see this got changed to take a boolean payload, but it doesn't look like we're dispatching `setMessagesLoaded(false)` anywhere - it gets set as `state.messagesLoaded = false` above.  Do we anticipate dispatching that later?

Reply:
Oh, that's fair. I was confused by the fact that we had an action with the same name as the state (`messagesLoaded` and `messagesLoaded`). I wanted to use the state in my "do we need to refetch?" check again so I didn't want them to clash, so I renamed it to `setMessagesLoaded` but then it seemed weird not to accept a value. I don't feel strongly about this though TBH>